packages/common/core-utils/src/test/events/events.spec.ts

```diff
@@ -151,11 +151,14 @@ describe("EventEmitter", () => {
 	it("errors on multiple registrations of the same listener", () => {
 		const emitter = createEmitter<TestEvents>();
 		let count = 0;
-		const listener = () => (count += 1);
+		const listener = (): number => (count += 1);
 		emitter.on("open", listener);
 		assert.throws(
 			() => emitter.on("open", listener),
-			(e: Error) => validateAssertionError(e, /register.*twice.*open/),
+			// (e: Error) => validateAssertionError(e, /register.*twice.*open/),
+			(e: Error) => {
+				throw new Error(`${e} - /register.*twice.*open/`);
```

Comment:
This seems wrong, the second argument of assert.throws should be confirming that the specific error thrown is the right error.

Reply:
I split the tests between the tree and core-utils packages. The two tests needing `validateAssertionError` are moved to tree, while the remaining tests were placed in core-utils.